projects/1.18/assets/alexs-cloud-storage/cloudstorage/lang/zh_cn.json

```
@@ -0,0 +1,112 @@
+  "entity.minecraft.villager.cloudstorage.balloon_salesman": "气球推销员",
+  "item.cloudstorage.spawn_egg_badloon": "气球怪刷怪蛋",
+  "item.cloudstorage.spawn_egg_bloviator": "Bloviator刷怪蛋",
+  "item.cloudstorage.balloon_bit": "气球碎块",
+  "item.cloudstorage.guide_book": "云存储与你",
+  "item.cloudstorage.guide_book.desc": "Alex Khaan 博士教授 著",
+  "item.cloudstorage.balloon_inventory": "气球",
+  "item.cloudstorage.balloon": "气球",
+  "item.cloudstorage.balloon.static": "带静电",
+  "item.cloudstorage.loot_balloon": "战利品气球",
+  "item.cloudstorage.propeller_hat": "螺旋桨帽",
+  "item.cloudstorage.angry_cloud_in_a_bottle": "愤怒云瓶",
+  "item.cloudstorage.happy_cloud_in_a_bottle": "欢乐云瓶",
+  "item.cloudstorage.propeller_hat": "螺旋桨帽",
+  "item.cloudstorage.balloon_buddy_inventory": "气球伙伴",
+  "item.cloudstorage.balloon_buddy": "气球伙伴",
+  "item.cloudstorage.balloon_arrow": "气球箭",
+  "item.cloudstorage.cotton_candy": "棉花糖",
+  "item.cloudstorage.music_disc_drift": "音乐唱片",
+  "item.cloudstorage.music_disc_drift.desc": "Shroomy - Drift",
+  "item.cloudstorage.sky_temple_map": "天空神庙探险家地图",
+  "item.cloudstorage.big_balloon_map": "大型气球探险家地图",
+  "block.cloudstorage.cloud": "云块",
+  "block.cloudstorage.static_cloud": "带静电的云块",
+  "block.cloudstorage.cloud_chest": "云箱",
+  "block.cloudstorage.static_cloud_chest": "带静电的云箱",
+  "block.cloudstorage.balloon_stand": "气球摊",
+  "cloudstorage.container.cloud_chest": "云箱",
+  "cloudstorage.container.static_cloud_chest": "带静电的云箱",
+  "cloudstorage.container.cloud_chest.button_0": "背包",
+  "cloudstorage.container.cloud_chest.button_1": "整理容器",
+  "cloudstorage.container.cloud_chest.button_2": "搜索",
+  "cloudstorage.container.cloud_chest.searchbar": "搜索：",
+  "cloudstorage.container.balloon_stand": "气球摊",
+  "cloudstorage.container.balloon_stand.color": "颜色：%s",
+  "cloudstorage.container.balloon_stand.slots": "%s / %s 个槽位",
+  "cloudstorage.container.balloon_stand.private_slots": "云：",
+  "cloudstorage.container.balloon_stand.public_slots": "带静电的云：",
+  "message.cloudstorage.no_sky_access": "云箱必须放在高空之下",
+  "message.cloudstorage.no_balloon": "这个云箱没有指定的气球",
+  "message.cloudstorage.no_inventory_space": "这种气球颜色没有对应的云背包",
+  "message.cloudstorage.no_static": "云箱不能挂上带静电的气球",
+  "death.attack.sneak_balloon_attack": "%s 被一只鬼鬼祟祟的气球怪杀死了",
+  "advancements.cloudstorage.root.title": "云储存",
+  "advancements.cloudstorage.root.desc": "天空即为极限",
+  "advancements.cloudstorage.badloon.title": "放气",
+  "advancements.cloudstorage.badloon.desc": "在一只气球怪击破你之前击破它",
+  "advancements.cloudstorage.bloviator.title": "§mSky of Beginning§r 烦人的天空",
+  "advancements.cloudstorage.bloviator.desc": "杀死Bloviator以免你坠入死亡",
+  "advancements.cloudstorage.balloon.title": "不可击破",
+  "advancements.cloudstorage.balloon.desc": "用气球碎块和线来制造一个气球",
+  "advancements.cloudstorage.angry_cloud_in_a_bottle.title": "永远封印",
+  "advancements.cloudstorage.angry_cloud_in_a_bottle.desc": "把一只微小的Bloviator抓进瓶子里",
+  "advancements.cloudstorage.happy_cloud_in_a_bottle.title": "如果你给云朵一片饼干",
```

```suggestion
  "advancements.cloudstorage.happy_cloud_in_a_bottle.title": "如果你给云朵吃饼干",
  ```
  [If You Give a Mouse a Cookie](https://www.imdb.com/title/tt7344644/)
  原文：**If You Give a Cloud a Cookie**
  | 严重程度 |
  |--------------|
  | ℹ️ 信息 / 不阻止合并  |

---

ok

---

（因为提示还是不行，正在摸索中）